src/IssueViz.Security.UnitTests/OpenInIDE/Api/OpenInIDERequestHandlerTests.cs

```diff
@@ -237,17 +243,41 @@ public async Task ShowHotspot_DataIsValid_NavigationFailed_IssueStillAddedToStor
             SetConversionResponse(ValidServerHotspot, hotspotViz);
             SetNavigationRespone(hotspotViz, false);
             SetStoreExpectedItem(hotspotViz);
+            SetSelectionServiceExpectedItem(hotspotViz);
 
             // Act
             await testSubject.ShowHotspotAsync(ValidRequest)
                 .ConfigureAwait(false);
 
             CheckInfoBarCleared();
             CheckInfoBarShown();
-            CheckCalled(toolWindowService, stateValidatorMock, serverMock, converterMock, navigatorMock, storeMock);
+            CheckCalled(toolWindowService, stateValidatorMock, serverMock, converterMock, navigatorMock, storeMock, selectionServiceMock);
             logger.AssertPartialOutputStringExists(hotspotFilePath, hotspotStartLine.ToString());
         }
 
+        [TestMethod]
+        public async Task ShowHotspot_DataIsValid_IssueFirstAddedToStoreAndThenSelected()
```

Comment:
I'm not a fan of this.
I don't know if it's better to use instead something like `IHotspotsStore.AddAndSelect`.

Reply:
Yep; depends on what the overall selection/sync big picture looks like.